win32/Makefile

```diff
@@ -62,12 +62,12 @@ ifdef WIN32_RELOCATABLE
 endif
 
 # cygwin: built native w32 code
-NOCYGWIN=-mno-cygwin
+#NOCYGWIN=-mno-cygwin
```

Comment:
Why is this commented out?

Reply:
See https://ci.appveyor.com/project/chcg/aspell/builds/27190547

mingw32-make : g++: error: unrecognized command line option '-mno-cygwin'; did you mean '-mno-clwb'?
At line:1 char:1
+ mingw32-make
+ ~~~~~~~~~~~~

Reply:
See https://www.boost.org/doc/libs/1_31_0/tools/build/v1/gcc-nocygwin-tools.html

> This page describes the gcc-nocygwin toolset, which builds Boost using the -mno-cygwin option of the Cygwin gcc compiler. This avoids introducing dependencies on the Cygwin Unix-emulation layer, allowing you to build Windows executables which are not dependant on cygwin1.dll. If you're not worried about having the dependencies, you should probably be using the plain gcc toolset.
> 
> The other option for producing windows-native executables with gcc is the mingw version of gcc with the mingw toolset. However, if you're already using Cygwin, this toolset could save you having to install an additional compiler.
> 

, so this seems to be a cygwin specific compiler option, not available with mingw gcc